module-3/test/pop/HomePage.spec.js

```diff
@@ -1,3 +1,64 @@
-describe('HomePage Class', () => {
-    // TODO: write tests
-});
\ No newline at end of file
+const Browser = require('../mock/Browser');
+const ElementFinder = require('../mock/ElementFinder');
+const HomePage = require('../../pop/HomePage');
+const expect = require('chai').expect;
+
+describe.only('HomePage Class', () => {
+    beforeEach(() => {
+        global.element = ElementFinder.element;
+        global.browser = new Browser();
+    });
+    
+    afterEach(() => {
+        delete global.element;
+        delete global.browser;
+    });
+    
+    it('should be defined', () => {
+        expect(HomePage).to.be.instanceOf(Function);
+    });
+
+    it('should have the page name', () => {
+        const page = new HomePage('EPAM HomePage', 'http://epam.com', {css: 'body'});
```

Comment:
Can we have just one homepage?

Reply:
The "page" constant has been set once for all test cases.